src/node/Node.ts

```diff
@@ -173,7 +173,7 @@ export class Node extends EventEmitter {
         if (!this.manager.id) throw new Error('Don\'t connect a node when the library is not yet ready');
         if (this.destroyed) throw new Error('You can\'t re-use the same instance of a node once disconnected, please re-add the node again');
 
-        const resume = this.initialized && (this.manager.options.resume && this.manager.options.resumeKey);
+        const resume = this.initialized && (this.manager.options.resume && this.manager.options.sessionId);
```

Comment:
This needs to be reimplemented, since the sessionId will now come from this class. Could as simple as 
```js
const resume = this.initialized && (this.manager.options.resume && this.manager.options.resumeKey);
this.state = State.CONNECTING;
let headers: ResumableHeaders|NonResumableHeaders;
if (resume) {}
```
to
```js
if (this.sessionId) {}
```